browser/importer/brave_external_process_importer_client.cc

```diff
@@ -5,8 +5,11 @@
 
 #include "brave/browser/importer/brave_external_process_importer_client.h"
 
+#include <utility>
+
 #include "base/bind.h"
 #include "brave/browser/importer/brave_in_process_importer_bridge.h"
+#include "chrome/browser/service_sandbox_type.h"
```

Comment:
this doesn't seem right, what is this include needed for?

Reply:
I see `ProfileImport` is specifically in service_sandbox_type vs the general content/public/browser/sandbox_type.h header